mail/mail-server/Portfile

```diff
@@ -0,0 +1,877 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               active_variants 1.1
+
+name                    mail-server
+version                 1.0
+revision                0
+categories              mail net
+platforms               darwin
+supported_archs         noarch
+maintainers             {ieee.org:s.t.smith @essandess} openmaintainer
+license                 GPL-3
+distfiles
+
+description             Mail server configuration
+long_description ${description} \
+    Mail server working configuration that provides a basic, working, \
+    easily modifiable mail server. The configuration is built using \
+    postfix for the MTA, dovecot for the MDA, solr for fast search, \
+    rspamd for a milter, and clamav for email virus scans. The \
+    configuration includes a surrogate TLS certificate and DKIM.
+
+homepage                https://www.postfix.org/ \
+                        https://www.dovecot.org/ \
+                        https://rspamd.com/
```

Comment:
It does not really make sense to have more than one `homepage`. The whole point, AFAIK, is a landing place for the command:

`port gohome mailserver`

Please only use one. I'm not sure what it does in this case.

Reply:
Thanks. I just chose postfix. Done.